{{cookiecutter.project_name}}/pyproject.toml

```
@@ -2,7 +2,9 @@
 {%- if cookiecutter.backend == "hatch" %}
 requires = [
   "hatchling",
-  "hatch-vcs"
+{%- if cookiecutter.backend == "hatch" and cookiecutter.vcs == true %}
```

Already inside the `if cookiecutter.backend == "hatch"` block
```suggestion
{%- if cookiecutter.vcs == true %}
```